modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -175,7 +173,7 @@ TEST(TestAgeGenderIE, InferBasicImage)
 }
 
 struct ROIList: public ::testing::Test {
-    cv::gapi::ie::detail::ParamDesc params;
+    std::string model_path   = "", weights_path = "", device_id    = "";
```

Comment:
Such kind of initialization isn't necessary, because `std::string` has default `ctor`

Reply:
Oh, OK, thanks! removed